Source/core/NodeId.h

```diff
@@ -269,6 +261,8 @@ namespace Core {
         bool IsMulticast() const;
         uint8_t DefaultMask() const;
 
+        uint32_t EnablePermission() const;
```

Comment:
The NodeId is a data carrier object and should not have active methods, only carry/translate data. Permission setting is an active method that should from design perspective not be on a data carrier object. Suggest to "expose" the Group that might now be passed on the node (so a *const string& Group() const;* method) and leave the setting of the permissions up to new AccessControl.h class in the specific implementations (doorbel, socketport and others)

Reply:
done